README.md

```diff
@@ -36,6 +36,33 @@
 
 ![Query Graphs](resources/Graph.gif)
 
+- Configure the user setting `mongocosmosDB.graph.viewSettings` to customize which properties to display and which colors to use based on vertex label.
+```javascript
+    "cosmosDB.graph.viewSettings": {
+        "default": {
+            "vertices": [
+                {

```

Comment:
Label (poor name) is the "type" of node e.g. "person" or "company".  So it won't generally help distinguish between nodes.  Portal shows the ID by default, I think it makes sense for us as well (since label and ID are always there, only one can be in the default).

Reply:
Although an interesting option would be to allow showing the label in parentheses.  I wanted to allow expressions like "${name} (${label})", but there are semantics issues to sort out, and I figured I could add that later with a different property like "displayText" so I postponed it.